util/get-supported-module-env.js

```diff
@@ -4,95 +4,82 @@ const path = require('path');
 const fetch = (...args) => import('node-fetch').then(({default: fetch}) => fetch(...args));
 
-const URL = 'https://raw.githubusercontent.com/silverstripe/supported-modules/gh-pages/modules.json';
+/**
+ * Gets information for any "supported-module" repositories.
+ */
+async function getRelevantModules(jsonUrl, foundModules) {
+  const response = await fetch(jsonUrl);
+  const currentModules = await response.json();
+  // Only list supported-module, since the CMS 4 list includes some things we don't control or care about with other types.
+  const relevant = currentModules.filter(module => module.type === 'supported-module');
+  for (const module of relevant) {
+    foundModules[module.github] = module;
+  }
+}
 
-// Modules which are part of the default download
-const coreRepos = [
-  'silverstripe/recipe-cms',
-  'silverstripe/recipe-core',
-  'silverstripe/recipe-plugin',
-  'silverstripe/silverstripe-admin',
-  'silverstripe/silverstripe-asset-admin',
-  'silverstripe/silverstripe-assets',
-  'silverstripe/silverstripe-campaign-admin',
-  'silverstripe/silverstripe-cms',
-  'silverstripe/silverstripe-config',
-  'silverstripe/silverstripe-errorpage',
-  'silverstripe/silverstripe-framework',
-  'silverstripe/silverstripe-graphql',
-  'silverstripe/silverstripe-installer',
-  'silverstripe/silverstripe-reports',
-  'silverstripe-themes/silverstripe-simple',
-  'silverstripe/silverstripe-siteconfig',
-  'silverstripe/silverstripe-versioned',
-  'silverstripe/silverstripe-versioned-admin'
-];
+async function rebuildReposList() {
+  // Initialise this with repos that aren't in the supported-modules json
+  // Or which will otherwise be filtered out due to their type
+  const extraRepos = [
+    // GitHub actions
+    'silverstripe/gha-action-ci',
+    'silverstripe/gha-auto-tag',
+    'silverstripe/gha-ci',
+    'silverstripe/gha-dispatch-ci',
+    'silverstripe/gha-gauge-release',
+    'silverstripe/gha-generate-matrix',
+    'silverstripe/gha-issue',
+    'silverstripe/gha-keepalive',
+    'silverstripe/gha-merge-up',
+    'silverstripe/gha-pull-request',
+    'silverstripe/gha-run-tests',
+    'silverstripe/gha-tag-release',
+    'silverstripe/gha-trigger-ci',
+    'silverstripe/gha-update-js',
+    // tooling
+    'silverstripe/cow',
+    'silverstripe/rhino',
+    'silverstripe/github-issue-search-client',
+    'silverstripe/module-standardiser',
+    'silverstripe/silverstripe-tx-translator',
+    // extra bits and bobs
+    'silverstripe/.github',
+    'silverstripe/eslint-config',
+    'silverstripe/webpack-config',
+    'silverstripe/silverstripe-module',
+    'silverstripe/api.silverstripe.org',
+    'silverstripe/doc.silverstripe.org',
+    'silverstripe/silverstripe-userhelp-content',
+    'silverstripe/demo.silverstripe.org',
+    'silverstripe/silverstripe-frameworktest',
+  ];
+
+  const allModules = {};
+  for (const ref of extraRepos) {
+    allModules[ref] = { github: ref };
+  }
```

Comment:
```suggestion
  const allModules = {};
  for (let repo of extraRepos) {
    allModules[repo] = { github: repo };
  }
```

Reply:
Const is correct. I'm happy to change `ref` to `repo` though.

Reply:
Changed ref to repo.